replication-api-impl/src/main/java/com/connexta/replication/api/impl/persistence/ReplicationItemManagerImpl.java

```diff
@@ -11,15 +11,15 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ditto.replication.api.impl.persistence;
+package com.connexta.replication.api.impl.persistence;
```

Comment:
❗️  These files should all be moved to their proper directories to match the new package names.

Reply:
There have been some more packages/directory changes since this comment but it looks like the directories match the packages right now.